### ISSUE TITLE
docker container fails to mount

### ISSUE THREAD

**Issue by andrewmilkowski** [kbastani/docker-neo4j#4]
Hi, I did submit below to main docker github for evaluation, maybe it has to do with OS/docker combination, appreciate your feedback and guide

environment

[amilkowski@localhost ~]$ uname -a
Linux localhost.localdomain 3.10.0-229.1.2.el7.x86_64 #1 SMP Thu Mar 26 09:10:25 CDT 2015 x86_64 x86_64 x86_64 GNU/Linux
[amilkowski@localhost ~]$ docker -v
Docker version 1.5.0-dev, build fc0329b/1.5.0

running:

Scientific Linux release 7.1 (Nitrogen)
cpe:/o:redhat:enterprise_linux:7.1:ga

which really is Centos 7

trying to do neo4j docker from https://github.com/kbastani/docker-neo4j

docker pull kbastani/docker-neo4j
docker run -d -p 7474:7474 -v /home/amilkowski/data:/opt/data --name graphdb kbastani/docker-neo4j
container exists..

[amilkowski@localhost ~]$ sudo docker run -d -p 7474:7474 -v /home/amilkowski/data:/opt/data --name graphdb kbastani/docker-neo4j
2b2d38d3cfa1519f17fe896f37e6f66a743895f5ddabf099bba1ac3eff16ed8d
[amilkowski@localhost ~]$ sudo docker ps -a
CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES
2b2d38d3cfa1 kbastani/docker-neo4j:latest "/etc/bootstrap.sh - 3 seconds ago Exited (0) Less than a second ago graphdb

[amilkowski@localhost ~]$

IF I remove host to container mount override container runs (-v /home/amilkowski/data:/opt/data) below

so it does look like the feature of docker "mounting volumes from host" might be broken.. (ref. http://docs.docker.com/userguide/dockervolumes/)

[amilkowski@localhost ~]$ sudo docker ps -a
CONTAINER ID IMAGE COMMAND CREATED STATUS PORTS NAMES
26e3d3bcd81d kbastani/docker-neo4j:latest "/etc/bootstrap.sh - 4 seconds ago Up 3 seconds 1337/tcp, 0.0.0.0:7474->7474/tcp graphdb

**Comment by kbastani** [comment]
After you run the container once, you would use a different command to start it back up.

    docker start graphdb

**Comment by kbastani** [comment]
I've run into the same issue with Neo4j 2.2+. It looks like mounting volumes from host using boot2docker is no longer possible in 2.2 due to a bug with VirtualBox.

I'm going to check in with the Neo4j folks and see whether or not they plan to fix this soon. There is a workaround that requires copying the graph.db directory from your host into a mounted volume on your Docker container. It's not ideal, so I'm hoping the bug gets fixed in Neo4j.

Reported issues I've found relating to the root problem:
- http://stackoverflow.com/questions/18370043/neo4j-server-in-virtualbox-unable-to-create-rrd-store
- https://groups.google.com/forum/#!searchin/neo4j/docker/neo4j/9mATBkX2YxU/shL87NlE-YQJ

**Comment by alph486** [comment]
@kbastani any luck recently?

**Comment by anentropic** [comment]
Neo4j fix has apparently been merged into 2.2 branch https://github.com/neo4j/neo4j/pull/4780

the relevant commit https://github.com/neo4j/neo4j/commit/038ff587d1355b45f4f1de2788c228b26c0d9d91 appears to be present in 2.2.3 release

**Comment by taimur38** [comment]
This seems to be happening again with docker 1.8 

Docker version 1.8.1, build d12ea79

``` 
Starting Neo4j Server console-mode...
Using additional JVM arguments:  -server -XX:+DisableExplicitGC -Dorg.neo4j.server.properties=conf/neo4j-server.properties -Djava.util.logging.config.file=conf/logging.properties -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled -XX:-OmitStackTraceInFastThrow -XX:hashCode=5 -Dneo4j.ext.udc.source=tarball

2015-08-31 22:05:14.837+0000 INFO  [API] Setting startup timeout to: 120000ms based on 120000
2015-08-31 22:05:15.232+0000 INFO  [API] Successfully shutdown Neo4j Server.```
